lib/stdext.mli

```diff
@@ -40,12 +40,32 @@ module Unix : sig
 end
 
 module Option : sig
+  val pp :
+    (Format.formatter -> 'a -> unit) -> Format.formatter -> 'a option -> unit
+
+  val equal : ('a -> 'a -> bool) -> 'a option -> 'a option -> bool
+
   val map : f:('a -> 'b) -> 'a option -> 'b option
 
   val bind : f:('a -> 'b option) -> 'a option -> 'b option
+
+  module O : sig
```

Comment:
I would call the module `Monad_infix` or `Infix`

Reply:
The `O` stands for `Operators`, it's a convention I got from dune IIRC, I like it as it can be used for any kind of operators, not only infix ones.

Reply:
Alright!